packages/analyticsdx-template-lint/src/schemas/readiness-schema.json

```diff
@@ -150,25 +150,29 @@
             "OrgDatasetRowCount",
             "AppDatasetRowCount",
             "DataCloudRowCount",
+            "ApexCallout",
             "OrgPreferenceCheck",
-            "ApexCallout"
+            "AccessCheck"
           ],
           "enumDescriptions": [
             "A check against the number of objects in the org.",
             "A check against the number of rows in a dataset in the org",
             "A check against the number of rows in a dataset that will be created by this template in the app",
             "A check against the number of Data Cloud objects in the org. Only available in Winter '24 release.",
+            "A check using a method on the template's apexCallback class.",
             "",
-            "A check using a method on the template's apexCallback class."
+            ""
```

Comment:
qq is the reason we have "" for the accessCheck description that it's an internal only feature?

Reply:
Yep